scripts/perf.conf

```diff
@@ -8,8 +8,8 @@ avg_update =
 [netperf]
 result_file_pattern = .*.RHS
 ignore_col = 2
-avg_update = 4,2,3|12,5,10|13,6,11
-desc = The tests are *%s* seconds sessions of 'Netperf'. 'throughput' was taken from netperf's report.\nOther measurements were taken on the host.\nHow to read the results:\n - The Throughput is measured in Mbit/sec.\n - io_exit: io exits of KVM.\n - irq_inj: irq injections of KVM.\n
+avg_update = 
```

Comment:
It's there any reason to empty it?

Reply:
I just update it to "avg_update = 4,2,3" since this is no other values in the column need to be calculated for the average.